deploy/core/version.json

```diff
@@ -1 +1 @@
-{"version":"0.7.2","atom-shell":"0.19.5"}
+{"version":"0.7.2","electron":"0.19.5"}
```

Comment:
Should the electron version correspond with the one in deploy/electron/Gruntfile.js ?